src/controllers/index.controller.js

```diff
@@ -1,11 +1,12 @@
-let indexController = (templateModel) => {
-  let index = (req, res, next) => {
-    res.render('index', templateModel)
-  }
+const HttpStatus = require('http-status-codes')
+
+module.exports = () => {
+  const index = (req, res) =>
+    res.status(HttpStatus.OK).json({
+      message: 'success'
```

Comment:
Our index page should point to our Swagger main page once we merge the swagger branch.